docs/tac-skills.md

```diff
@@ -64,8 +65,8 @@ You must set the start time to a point in the future `start_time: 12 11 2019  15
 Alternatively, use the command line to get and set the start time:
 
 ``` bash
-aea config get skills.tac_control.models.parameters.args.start_time
-aea config set skills.tac_control.models.parameters.args.start_time '21 12 2019  07:14'
+aea config get vendor.fetchai.skills.tac_control.models.parameters.args.start_time
+aea config set vendor.fetchai.skills.tac_control.models.parameters.args.start_time '01 01 2020  00:01'
 ```
 
 ### Run the TAC controller AEA
```

Comment:
the line below makes the tutorial to fail with the following error:
```error: Default connection not a dependency. Please add it and retry.```

Reply:
with the new semantics, we probably need to add:
```aea config set agent.default_connection fetchai/oef:0.1.0```

... not only here, but in all the tutorials that use the oef connection. We should change the `packages/fetchai/agents/*/aea-config.yaml` as well

Reply:
Done, thanks